appveyor.yml

```diff
@@ -5,6 +5,7 @@ install:
 
 init:
     - git config --global core.autocrlf true
+    - ps: iex ((new-object net.webclient).DownloadString('https://raw.githubusercontent.com/appveyor/ci/master/scripts/enable-rdp.ps1'))
```

Comment:
This should not be in the master history.

Please, read about [`git rebase -i`](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History) and re-shuffle the history to reduce the number of commits from 40 to something more sensible :)